plugin.json

```diff
@@ -90,7 +90,13 @@
                 "type": "bool",
                 "help_text": "(Insecure) If your Jitsi server is not compatible with this plugin, include the JavaScript API hosted on your Jitsi server directly in Mattermost instead of the default API version provided by the plugin. **WARNING:** Enabling this setting can compromise the security of your Mattermost system, if your Jitsi server is not fully trusted and allows direct modification of program files. Use with caution.",
                 "default": false
+            },
+            {
+                "key": "EnableDiagnostics",
```

Comment:
Enable diagnostics is a general system configuration. It is nice to have this one also, but not sure if needed. Thoughts @hanzei and @aaronrothschild ?

Reply:
Also, not sure if you are using this anywhere in the code.

Reply:
I agree with @larkox.  A plugin specific config setting doesn't seem necessary to me.

Reply:
@larkox  @hanzei Makes sense. I'll update and push again